tools/deployment-cli-tools/ch_cli_tools/helm.py

```diff
@@ -410,7 +410,7 @@ def image_tag(self, image_name, build_context_path=None, dependencies=()):
             ignore = set(DEFAULT_IGNORE)
             if os.path.exists(ignore_path):
                 with open(ignore_path) as f:
-                    ignore = ignore.union({line.strip() for line in f})
+                    ignore = ignore.union({line.strip() for line in f if line.strip() and not line.startswith('#')})
```

Comment:
**Suggestion**

`union` will work with any iterable, so you can remove the braces to not assign to a set first.